canoe/bindings/src/lib.rs

```diff
@@ -92,8 +92,24 @@ sol! {
         uint32[][] nonSignerStakeIndices;
     }
 
+    struct EigenDACertV3 {
+        BatchHeaderV2 batchHeaderV2;
+        BlobInclusionInfo blobInclusionInfo;
+        NonSignerStakesAndSignature nonSignerStakesAndSignature;
+        bytes signedQuorumNumbers;
+    }
+
+    #[derive(PartialEq)]
+    enum StatusCode {
```

Comment:
these are CertVerifierV3 error codes right? Like V4 could have new error codes I think? Prob should name `StatusCodeCVV3` or something?
Also what about generating these bindings using alloy? Is that too hard/annoying?